src/macros.rs

```diff
@@ -1,4 +1,4 @@
-///! Utility macros
+//! Utility macros
```

Comment:
Not sure why clippy started acting up with this now. The lint was introduced on 1.70 and latest commit is the one enabling clippy for the pipelines.